actionview/lib/action_view/base.rb

```diff
@@ -162,6 +162,9 @@ class Base
     # Specify whether submit_tag should automatically disable on click
     cattr_accessor :automatically_disable_submit_tag, default: true
 
+    # Specify whether the view rendering includes which partial produced the output
+    cattr_accessor :render_hints, default: false
```

Comment:
You don't need this being set here since we are not using it anywhere.

Reply:
The Railtie initializer "action_view.set_configs" fails if this doesn't exist.
`actionview/lib/action_view/railtie.rb:69`